client/src/components/Brand.tsx

```diff
@@ -1,42 +1,67 @@
 import { Link } from 'react-router-dom';
+import { useContext } from 'react';
+import { closeContext } from './NavDrawerCloseContext';
 type BrandProps = {
   isOpen: boolean;
+  handleClosing: () => void;
 };
 
-export function Brand({ isOpen }: BrandProps) {
+export function Brand({ isOpen, handleClosing }: BrandProps) {
+  const { closeNavDrawer } = useContext(closeContext);
+
   return (
     <>
       {isOpen && (
-        <ul className="ml-3 cursor-pointer">
+        <ul className="ml-3 cursor-pointer" onClick={closeNavDrawer}>
```

Comment:
I think this should be `handleClosing` instead of `closeNavDrawer`. Then you won't need `closeNavDrawer` anymore.